<launch>
  <arg name="pose_topic" default="/ekf_pose"/>
  <arg name="center_latitude" default="37.58" />
  <arg name="center_longitude" default="126.89" />
  <arg name="center_altitude" default="0.0" />
    <node name="hdmap" pkg="hdmap_localizer" type="hdmap_localizer.py" clear_params="true">
      <param name="pose_topic" value="$(arg pose_topic)"/>
      <param name="center_latitude" value="$(arg center_latitude)" />
      <param name="center_longitude" value="$(arg center_longitude)" />
      <param name="center_altitude" value="$(arg center_altitude)" />
    </node>
</launch>
